tests/smoketest.py

```diff
@@ -43,45 +42,58 @@ def run(command, runner=check_call, **kwargs):
     run("git clone --depth 1 --recurse-submodules https://github.com/HumanCellAtlas/data-store-cli")
 
 run("http --check-status https://${API_HOST}/v1/swagger.json > data-store-cli/swagger.json")
-run("pip install -r data-store-cli/requirements.txt")
-run("python -c 'import sys, hca.dss.regenerate_api as r; r.generate_python_bindings(sys.argv[1])' swagger.json",
-    cwd="data-store-cli")
-run("find data-store-cli/hca -name '*.pyc' -delete")
-run("pip install --upgrade --no-deps .", cwd="data-store-cli")
+venv = tempfile.mkdtemp()
```

Comment:
use `with tempfile.TemporaryDirectory() as venv:`

it provides cleanup for free.

Reply:
Will do.